### PR TITLE
Fix Last Update

### PR DESCRIPTION
**Pull request by nafiz1001**

### CODE REVIEW

**Comment by nafiz1001** [comment]
image

File: frontend/webpack.config.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ module.exports = (env, argv) => ({
     new webpack.DefinePlugin({
       GIT_COMMITHASH: JSON.stringify(gitRevisionPlugin.commithash()),
       GIT_BRANCH: JSON.stringify(gitRevisionPlugin.branch()),
-      GIT_LASTUPDATE: JSON.stringify(child_process.execSync('git log -1 --format=%cI').toString().trim()),
+      GIT_LASTUPDATE: JSON.stringify(child_process.execSync('git log -1 --format=%cD').toString().trim()),
```
Comment (UlysseFG) [inline frontend/webpack.config.js:63]: i like much more the iso format ... %ci          
You could also cut off after the date to keep it simple.

**Comment by nafiz1001** [comment]
How about this?
image

**Comment by UlysseFG** [comment]
Only worry is that the time might not be from the right timezone.

**Pull request merged**